### PR TITLE
bundle: add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by andrik**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages.pt-BR/common/bundle.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# bundle
+
+> Gerenciador de dependências da linguagem de programação Ruby.
+
+- Instalar todas as gemas definidas no Gemfile:
+
+`bundle install`
+
+- Atualizar todas as gemas, respeitando as regras definidas no Gemfile, e recriar o arquivo Gemfile.lock:
```
[the remark this answers is not in the document]
Comment (andrik) [reply pages.pt-BR/common/bundle.md]: Eu imaginei que mandarias esse feedback :smile: 
Nesse caso, acho que fica melhor deixar após o 'Atualizar...' pois as regras valem apenas para a atualização e não para a recriação do Gemfile
O que achas?

[the remark this answers is not in the document]
Comment (schneiderl) [reply pages.pt-BR/common/bundle.md]: Tenho que concordar. Parece fazer sentido visto que as regras se aplicam a atualização.

Obrigado pela Pull Request. Vou aprovar as alterações. 👍

**Comment by waldyrious** [comment]
The translation itself looks fine to me, but I'd suggest wrapping `Gemfile` and `Gemfile.lock` in backticks, since they're specific filenames. If you're feeling generous, a similar update to the English page (in this same PR) would be appreciated :)

**Comment by andrik** [comment]
@waldyrious You're right, I made the changes and i'll fix the English page too

**Comment by waldyrious** [comment]
For future reference, the changes to the English page were done in #2894.

**Pull request merged**